<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interwuziqi</class>
 <widget class="QDialog" name="interwuziqi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>interwuziqi</string>
  </property>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>590</y>
     <width>160</width>
     <height>29</height>
    </rect>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>480</y>
     <width>150</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="internetText">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>未连接网络</string>
   </property>
  </widget>
  <widget class="QLabel" name="prompt">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>50</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>游戏还未开始</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="prompt_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>590</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>跳转到作者主页</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>590</y>
     <width>67</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>音量大小</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>180</y>
     <width>150</width>
     <height>85</height>
    </rect>
   </property>
   <property name="text">
    <string>准备</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>580</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>制作者信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playMusic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>播放音乐</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
